config/webpack.development.config.js

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Please, add this rule to your configuration file instead of writing it at the top of each file

Reply:
in this case, to my mind, it's okey, 
we need to use 'require' in webpack config or add a plugin like this
https://stackoverflow.com/questions/31903692/how-can-i-use-es6-in-webpack-config-js

add a rule to give us the possibility to use 'require' in another part of the project it's not okey